neo4j-cli/aura/internal/subcommands/import/job/get_test.go

```diff
@@ -0,0 +1,564 @@
+)
+
+func TestGetImportJobByIdWithoutProgress(t *testing.T) {
+	helper := testutils.NewAuraTestHelper(t)
+	defer helper.Close()
+
+	mockHandler := helper.NewRequestHandlerMock("/v2beta1/projects/f607bebe-0cc0-4166-b60c-b4eed69ee7ee/import/jobs/87d485b4-73fc-4a7f-bb03-720f4672947e", http.StatusOK, `
+{
+    "data": {
+        "id": "87d485b4-73fc-4a7f-bb03-720f4672947e",
+        "import_type": "cloud",
+        "info": {
+            "state": "Completed",
+            "start_time": "2025-08-15T13:12:51Z",
+            "completion_time": "2025-08-15T13:15:19Z",
+            "exit_status": {
+                "state": "Cancelled",
+                "message": "Cancelled"
+            },
+            "cancellation_requested_time": "2025-08-15T13:15:19.655209Z",
+            "submitted_time": "2025-08-15T13:12:50.499797Z",
+            "last_update_time": "2025-08-18T12:25:53.469873Z",
+            "percentage_complete": 95.23
+        },
+        "data_source": {
+            "id": "177216b3-49b9-4e24-a414-a9ecbb448c53",
+            "type": "postgresql",
+            "name": "AWS_POSTGRES_FLIGHTS"
+        },
+        "aura_target": {
+            "db_id": "07e49cf5",
+            "project_id": "f607bebe-0cc0-4166-b60c-b4eed69ee7ee"
+        },
+        "user_id": "f607bebe-0cc0-4166-b60c-b4eed69ee7ee"
+    }
+}
+	`)
+
+	helper.SetConfigValue("aura.beta-enabled", true)
+
+	helper.ExecuteCommand("import job get --project-id=f607bebe-0cc0-4166-b60c-b4eed69ee7ee 87d485b4-73fc-4a7f-bb03-720f4672947e")
+
+	mockHandler.AssertCalledTimes(1)
+	mockHandler.AssertCalledWithMethod(http.MethodGet)
+
+	helper.AssertErr("")
+	helper.AssertOutJson(`
+{
+    "data": {
+        "aura_target": {
+            "db_id": "07e49cf5",
+            "project_id": "f607bebe-0cc0-4166-b60c-b4eed69ee7ee"
+        },
+        "data_source": {
+            "id": "177216b3-49b9-4e24-a414-a9ecbb448c53",
+            "name": "AWS_POSTGRES_FLIGHTS",
+            "type": "postgresql"
+        },
+        "id": "87d485b4-73fc-4a7f-bb03-720f4672947e",
+        "import_type": "cloud",
+        "info": {
+            "cancellation_requested_time": "2025-08-15T13:15:19.655209Z",
+            "completion_time": "2025-08-15T13:15:19Z",
+            "exit_status": {
+                "message": "Cancelled",
+                "state": "Cancelled"
+            },
+            "last_update_time": "2025-08-18T12:25:53.469873Z",
+            "percentage_complete": 95.23,
+            "start_time": "2025-08-15T13:12:51Z",
+            "state": "Completed",
+            "submitted_time": "2025-08-15T13:12:50.499797Z"
+        },
+        "user_id": "f607bebe-0cc0-4166-b60c-b4eed69ee7ee"
+    }
+}
+	`)
+}
+
+func TestGetImportJobByIdWithProgress(t *testing.T) {
+	helper := testutils.NewAuraTestHelper(t)
+	defer helper.Close()
+
+	mockHandler := helper.NewRequestHandlerMock("/v2beta1/projects/f607bebe-0cc0-4166-b60c-b4eed69ee7ee/import/jobs/87d485b4-73fc-4a7f-bb03-720f4672947e", http.StatusOK, `
```

Comment:
Is this mock checking the query param `progress`?

`--progress` should be adding that queryParam to the request no?

Reply:
@angrykoala - discussed offline and we figured out we should use `AssertCalledWithQueryParam`.